tests/scripts/perf-prep.sh

```diff
@@ -57,22 +63,38 @@ unzip -q -o benchview.zip -d ./tests/scripts/Microsoft.BenchView.JSONFormat
 python3.5 --version
 python3.5 ./tests/scripts/Microsoft.BenchView.JSONFormat/tools/machinedata.py
 
-# Set up the copies
-# Coreclr build containing the tests and mscorlib
-curl https://ci.dot.net/job/$perfBranch/job/master/job/release_windows_nt/lastSuccessfulBuild/artifact/bin/tests/tests.zip -o tests.zip
+if [ $throughput -eq 1]; then
+    # Clone corefx
+    if [ -d "_" ]; then
+        rm -r -f _
+    fi
+    mkdir _
+    git clone https://github.com/dotnet/corefx.git _/fx
+    cd _/fx
+
+    # Checkout the specific commit we want
+    git checkout f0b9e238c08f62a1db90ef0378980ac771204d35
```

Comment:
I locked on it because we don't want a moving testing target in addition to a moving jit. It just happened to be the commit that we were at when I was testing. I'm open to choosing a different commit (maybe some release commit?)